### PR TITLE
Make the "MOUNT_POINT" docker env configurable

### PR DESCRIPTION
**Pull request by cheyang**

### CODE REVIEW

File: integration/docker/entrypoint.sh [hunk 1]
```diff
@@ -15,7 +15,6 @@ set -e
 ALLUXIO_HOME="/opt/alluxio"
 NO_FORMAT='--no-format'
 FUSE_OPTS='--fuse-opts'
-MOUNT_POINT='/mnt/alluxio-fuse'
```
Comment (ZacBlanco) [inline integration/docker/entrypoint.sh:18]: I think we can do this in one line 

```suggestion
 MOUNT_POINT="${MOUNT_POINT:-/mnt/alluxio-fuse}"
```

Comment (cheyang) [reply integration/docker/entrypoint.sh:18]: Thanks for your suggestion. Done

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**